### PR TITLE
finshed with file viewer for saving and loading models

### LINKED ISSUE #135: Finish the GUI for the file view + saving and loading data
# Description
The file system is simply a QListView displaying a QFileSystemModel. Currently, you can enter directories and files (not correct behavior) and the top view shows the directory stack which allows a user to jump back to any outer directory. There is currently no button for saving and loading, NOTE: this GUI should both handle saving and loading since it is largely the same features just small number of changes. If you think it is worth if you could use inheritance to achieve this but I would just provide a boolean in the  constructor that changes the functionality.

### PR DESCRIPTION
**Pull request by Aleman778**
# Proposed changes
* Fixed file view so you can save and load files using the GUI
* Extra: Added some modal window functionality
* Extra: added scrolling to settings view

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

- Create some example process and save it using the GUI then close the program and using the GUI load the process and it should look exactly the same.

# Related issue
Fixes #135 

# Checkboxes
- [x] I have run the unit tests with `pytest`
- [x] I formatted the changes with `./scripts/formatter.sh`

### CODE REVIEW

File: src/main/python/untitled.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+filename: untitled
```
Comment (hugowangler) [inline src/main/python/untitled.yaml:1]: I think this file was accidentally commited.

**Review by hugowangler — CHANGES_REQUESTED** [review]

File: src/main/python/untitled.yaml (hunk 1 above)
Comment (hugowangler) [reply src/main/python/untitled.yaml:1]: Other than this it looks good and seems to be working!

**Review by hugowangler — APPROVED** [review]
looks good.

File: src/main/python/design_view.py [hunk 2]
```diff
@@ -80,17 +70,24 @@ def __init__(self, main_window, *args, **kwargs):
         self.cancel_actions_shortcut.activated.connect(self.cancel_actions)
 
     def save_model(self):
```
Comment (markhakansson) [inline src/main/python/design_view.py:72]: The naming scheme `save_model` etc. might be confused for the NLP models. Maybe we should use something more descriptive like "taskboard" or something? Your thoughts Alex?

**Review by markhakansson — CHANGES_REQUESTED** [review]

File: src/main/python/design_view.py (hunk 2 above)
Comment (Aleman778) [reply src/main/python/design_view.py:72]: Yeah, so far I have just called it a process but taskboard is not a bad idea. I was just using model as in data model.

**Review by markhakansson — APPROVED** [review]

**Pull request merged**